### ISSUE TITLE
Add pickup scheduling capability for LAW

### ISSUE THREAD

**Issue by saseestone** [sul-dlss/mylibrary#613]
After adding SearchWorks request links on the Law collection ([#2575](https://github.com/sul-dlss/SearchWorks/issues/2575) we need to add Law to the pickup drop down.

Targeting a release date of Monday, Nov. 9th.

Law plans to use their LibCal calendar.  They provided:
<script src="//ajax.googleapis.com/ajax/libs/jquery/3.4.0/jquery.min.js"></script>
<script>
jQuery.getScript("https://api3.libcal.com/js/myscheduler.min.js", function() {
    jQuery("#mysched_73555").LibCalMySched({iid: 3313, lid: 1122, gid: 11656, uid: 73555, width: 560, height: 680, title: 'Pickup at Outside of Law Library', domain: 'https://api3.libcal.com'});
});
</script>
 

<button id="mysched_73555" href="#">Pickup Requests</button>

<style>
#mysched_73555 {
  background: #B30000;
  border: 1px solid #B30000;
  border-radius: 4px;
  color: #FFFFFF;
  font: 14px Arial, Helvetica, Verdana;
  padding: 8px 20px;
  cursor: pointer;
}
#mysched_73555:hover, #mysched_73555:active, #mysched_73555:focus {
  opacity: 0.6;
}
</style>
